File: backend/src/main/java/edu/cit/mabini/meditrack/dto/MedicalRecordDto.java
package edu.cit.mabini.meditrack.medicalrecord;

import lombok.*;

import java.time.LocalDate;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class MedicalRecordDto {
    private Long id;
    private Long patientId;
    private String diagnosis;
    private String treatment;
    private String prescription;
    private String notes;
    private LocalDate visitDate;
}

File: mobile/app/src/main/java/edu/cit/mabini/meditrack/model/MedicalRecordDto.kt
package edu.cit.mabini.meditrack.model

data class MedicalRecordDto(
    val id: Long? = null,
    val patientId: Long,
    val diagnosis: String,
    val treatment: String,
    val prescription: String,
    val notes: String,
    val visitDate: String
)
